client/my-sites/plugins/style.scss

```diff
@@ -156,16 +156,22 @@ body.is-section-plugins:not( .is-nav-unification ) {
 	}
 }
 
+.plugin-details__tags {
+	position: absolute;
+	font-size: $font-body-extra-small;
+	color: var( --studio-gray-40 );
+	margin-top: 85px;
```

Comment:
When setting an absolute element it's suggested that we always declare at least `top, left`

Can we set the `padding-top: 100px` to a variable and use `top: calc( $padding-top - 15px)` here? That way those two implementations would be bound together and we could avoid problems due to tags being `absolute`.

Reply:
Added the suggestion [here](https://github.com/Automattic/wp-calypso/pull/58005/commits/57d99950207e524a5d9292b19256307140c5689c).